modules/serverless-rn-1-26-0.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies
+//
+// * /serverless/serverless-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="serverless-rn-1-26_{context}"]
+= Release notes for Red Hat {ServerlessProductName} 1.26
+
+{ServerlessProductName} 1.26 is now available. New features, changes, and known issues that pertain to {ServerlessProductName} on {product-title} are included in this topic.
+
+[id="new-features-1.26_{context}"]
+== New features
+
+* {ServerlessProductName} now uses Knative Serving 1.5.
+* {ServerlessProductName} now uses Knative Eventing 1.5.
+* {ServerlessProductName} now uses Kourier 1.5.
+* {ServerlessProductName} now uses Knative (`kn`) CLI 1.5.
+* {ServerlessProductName} now uses Knative Kafka 1.5.
+* {ServerlessProductName} now uses Knative Operator 1.3. FIXME please confirm this needs explicit listing here
```

Comment:
Yes. Let's mention it here. And we should also mention the new stuff that it brings. Namely the new `v1beta1` API for `operator.knative.dev`. The previous one was `v1alpha1`. This is for both KnativeEventing and KnativeServing custom resources:
```
apiVersion: operator.knative.dev/v1beta1
kind: KnativeEventing
```
...
```
apiVersion: operator.knative.dev/v1beta1
kind: KnativeServing
```
We should probably also change all occurrences in other Serverless docs from v1alpha1 and v1beta1. Do you want a separate JIRA for that?

Reply:
Update: There's already a JIRA for updating the other docs: https://issues.redhat.com/browse/SRVCOM-2105

Reply:
@mgencur Thanks, I've stated that `v1beta1` _replaces_ `v1alpha1`. Is it too strong a word?

https://github.com/openshift/openshift-docs/pull/53365/files#diff-42f0e2f286972d7ab015af5520b7dfcae1670d602d81e5cefbd5ddb488099653R41